ichnaea/tests/base.py

```diff
@@ -299,6 +300,10 @@ def cleanup_tables(cls, engine):
 
 class RedisIsolation(object):
 
+    def tearDown(self):
+        self.redis_client.flushdb()
+        super(RedisIsolation, self).tearDown()
+
```

Comment:
Ah, I decided against putting any of the standard setUp/tearDown methods on the *Isolation classes, as reasoning about what would get called with multiple inheritance gets really tricky. That's why anyone who subclasses from any of the isolation classes has to manually call the setup_/teardown_/cleanup methods provided on those isolation classes.

I'd rather keep it as it was. I mentioned the only required change at https://github.com/mozilla/ichnaea/pull/432/files#r29719552